src/app/loan/loan.component.html

```diff
@@ -1,4 +1,6 @@
-<td>{{book?.title}}</td>
-<td>{{book?.authors}}</td>
+<td *ngIf="isReservationList"><a href="loans">{{user?.firstName}} {{user?.lastName}}</a></td>
+<td><a href="book/{{book?.id}}">{{book?.title}}</a></td>
+<td *ngIf="!isReservationList">{{book?.authors}}</td>
 <td>{{loan?.startDate}}</td>
-<td *ngIf="loan?.endDate != null">{{loan?.endDate}}</td>
\ No newline at end of file
+<td *ngIf="loan?.endDate != null && !isReservationList">{{loan?.endDate}}</td>
+<td *ngIf="isReservationList"><button type="button" class="btn btn-success">Innemen</button></td>
```

Comment:
Is het handig om met deze ifs te doen of kan er een ander component voor gemaakt worden.
Voor de innemen knop die moet uiteindelijk ook te zien zijn op een profiel pagina, als de user die hem bekijkt een trainer is.

Reply:
Dit stem ik af met Ilias!